core/src/tpu.rs

```diff
@@ -87,6 +92,13 @@ impl Tpu {
             SigVerifyStage::new(packet_receiver, verified_sender, verifier)
         };
 
+        let (vote_verified_sender, vote_verified_receiver) = unbounded();
+
+        let vote_sigverify_stage = {
+            let verifier = TransactionSigVerifier::new_reject_non_vote();
+            SigVerifyStage::new(vote_packet_receiver, vote_verified_sender, verifier)
```

Comment:
Sigverify stage already uses 4 threads for verification, can we just half of those instead of adding 4 more?

Reply:
How constrained the resources are by adding 4 more threads. If necessary, perhaps can add a parameter to specify number of threads when creating sigverify stage.

Reply:
Agree that cli args are better, perhaps adding two args, one for number of threads for transaction sigverify, one for vote sigverify, after all, these two don't need to have same throughputs. Maybe in a separate PR?

Reply:
Separate PR sounds good

Reply:
The actual sigverify operation on CPU is done with a Rayon threadgroup, so the number of cores used is basically the same.

Reply:
Multiple threads helps with the GPU path. Also, the throughput does need to be basically the same, because you can get DOS'ed on any port and any port can deliver roughly the network bandwidth of the system.

Reply:
#20170